appium-cookies/test/cookies-specs.js

```diff
@@ -0,0 +1,78 @@
+// transpile:mocha
+
+import cookie from '../lib/cookies.js';
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import 'mochawait';
+import 'should';
+
+chai.should();
+chai.use(chaiAsPromised);
+chai.expect();
+let expect = chai.expect;
+
+describe(`cookies.js`, () => { 
+  it(`should properly read a cookie and return the cookie's value given the key`, () => {
+  	let cookieMan = new cookie(`k=v`);
+  	cookieMan.readCookie(`k`).should.equal(`v`);
+  });
+
+
+	it (`should return undefined for an undefined key value`, () => {
```

Comment:
this line and the following lines have odd indentation. are you sure you're not using tabs? also, we don't need a space after `it` in this and the following lines.